events/autocracy_events.txt

```diff
@@ -0,0 +1,321 @@
+		name = autocracy.1.b
+		add_loyalists = {
+			pop_type = aristocrats
+			value = 0.2
+		}
+		add_loyalists = {
+			pop_type = capitalists
+			value = 0.2
+		}
+		ig:ig_industrialists = {
+			remove_ideology = ideology_plutocratic
+			add_ideology = ideology_neo_feudalistic
+			set_variable = {
+				name = neo_feudalistic_leader_ideology_weights
+			}
+		}
+		custom_tooltip = neo_feudalistic_leader_ideology_weights_applied.tt
+		add_modifier = {
+			name = rise_of_neo_feudalism_mod
+			months = long_modifier_time
+		}
+	}
+	option = {
+		name = autocracy.1.c
+		add_loyalists = {
+			pop_type = clergymen
+			religion = root.religion
+			value = 0.2			#originally was 0.6 (!) The doc wanted it to be 0.05, but that feels too low... Especially for the designated religious option and after waiting 30 years.
+		}
+		add_loyalists = {
+			pop_type = farmers
+			religion = root.religion			#I am doing something fun here. Only the RELIGIOUS (following the only correct and true religion) farmers will become loyalists.
+			value = 0.05
+		}
+		add_loyalists = {
+			pop_type = laborers
+			religion = root.religion
+			value = 0.05
+		}
+		ig:ig_intelligentsia = {
+			remove_ideology = ideology_anti_clerical
+			remove_ideology = ideology_republican
+			add_ideology = ideology_neo_scholastic
+			set_variable = {
+				name = neo_scholastic_leader_ideology_weights
+			}
+		}
+		custom_tooltip = neo_scholastic_leader_ideology_weights_applied.tt
+		add_modifier = {
+			name = rise_of_neo_scholasticism_mod
+			months = long_modifier_time
+		}
+	}
+}
+
+# secret societies
+autocracy.2 = {
+	type = country_event
+	placement = scope:secret_societies_state
+	title = autocracy.2.t
+	desc = autocracy.2.d
+	flavor = autocracy.2.f
+	event_image = {
+		video = "votp_conspiring"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+	duration = 3
+	trigger = {
+		NOT = {
+			has_variable = secret_societies_timer
+		}
+		OR = {
+			has_law = law_type:law_monarchy
+			has_law = law_type:law_theocracy
+		}
+		has_law = law_type:law_state_religion
+		has_law = law_type:law_autocracy
+		ig:ig_intelligentsia = {
+			is_marginal = no
+			is_in_government = no
+		}
+		OR = {
+			has_law = law_type:law_censorship
+			has_law = law_type:law_outlawed_dissent
+			ig:ig_intelligentsia = {
+				is_being_suppressed = yes
+			}
+		}
+		any_scope_state = {
+			is_incorporated = yes
+			any_scope_pop = {
+				is_pop_type = academics
+			}
+		}
+	}
+	immediate = {
+		set_variable = {
+			name = secret_societies_timer
+			months = normal_modifier_time
+		}
+		ig:ig_intelligentsia = {
+			save_scope_as = secret_society_ig
+		}
+		random_scope_state = {
+			limit = {
+				is_incorporated = yes
+				any_scope_pop = {
+					is_pop_type = academics
+				}
+			}
+			save_scope_as = secret_societies_state
+			random_scope_pop = {
+				limit = {
+					is_pop_type = academics
+				}
+				save_scope_as = secret_society_pop
+			}
```

Comment:
Unused scope? also problematic scope because there probably will be multiple academic pops with slightly different religions and/or cultures.

Reply:
Part of the vanilla JE, not our work. I am fairly sure the scope is used by one of the two random events, the hidden societies one